clang/test/Driver/hexagon-toolchain-elf.c

```diff
@@ -401,6 +401,30 @@
 // CHECK338-NOT: "--end-group"
 // CHECK338: "{{.*}}/Inputs/hexagon_tree/Tools/bin/../target/hexagon/lib/v60/fini.o"
 
+// RUN: %clangxx -### -target hexagon-unknown-elf \
+// RUN:   -ccc-install-dir %S/Inputs/hexagon_tree/Tools/bin \
+// RUN:   -mcpu=hexagonv60 \
```

Comment:
These files wrap too quickly. Most tests pack more than one options on one line and don't start a new line just for `%s 2>&1`.